pkg/server/moon.go

```diff
@@ -0,0 +1,100 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/Lighty0410/ekadashi-server/pkg/mongo"
+)
+
+type MoonPhases struct {
+	Success bool                `json:"success"`
+	Err     error               `json:"error"`
+	Resp    []MoonPhaseResponse `json:"response"`
+}
+
+type MoonPhaseResponse struct {
+	Moon Moon `json:"moon"`
+	Sun  Sun  `json:"sun"`
+}
+
+type Sun struct {
+	RiseISO time.Time `json:"riseISO"`
+}
+
+type Moon struct {
+	Rise    int       `json:"rise"`
+	RiseISO time.Time `json:"riseISO"`
+	Phase   Phase     `json:"phase"`
+}
+
+type Phase struct {
```

Comment:
Do you think this can be embedded? Personally I see no reason for this to be a separate struct